.github/workflows/core-e2e-test.yml

```diff
@@ -0,0 +1,33 @@
+name: Core e2e tests
+on: [push, pull_request]
+jobs:
+  build:
+    name: Setup and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout cbioportal repo'
+        uses: actions/checkout@v2
+        with:
+          path: ./cbioportal
+      - name: 'Set up JDK 11'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: 'Cache Maven packages'
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: 'Build cBioPortal without authentication'
+        working-directory: ./cbioportal
+        run: |
+          mvn -q -U -Ppublic -DskipTests -Dauthenticate=false clean install
+      - name: 'Compare screenshots in the repo with screenshots of this portal loaded with the data from the amazon db'
+        working-directory: ./cbioportal
+        run: |
+          cd test/end-to-end && \
+          docker-compose up -d && \
+          sleep 30s && \
+          cd ../.. && \
+          bash test/end-to-end/test_make_screenshots.sh test/end-to-end/screenshots.yml
```

Comment:
this is very old - we don't need it any more since we have e2e tests in the frontend repo now